staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -258,18 +239,17 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 			unitCost := uint64(1)
 			lhs := args[0]
 			switch lhs.(type) {
-			case cel.Quantity:
-				return &unitCost
-			case cel.IP:
-				return &unitCost
-			case cel.CIDR:
-				return &unitCost
-			case *cel.Format: // Formats have a small max size.
-				return &unitCost
-			case cel.URL: // TODO: Computing the actual cost is expensive, and changing this would be a breaking change
+			case *cel.Quantity, cel.Quantity,
```

Comment:
For extension libraries you could use `cel.CostEstimatorOptions` and `cel.CostTrackerOptions` which would probably keep the maintenance of the associated extensions cleaner:

See: https://github.com/google/cel-go/blob/master/ext/sets.go#L104